pymilo/transporters/randomstate_transporter.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+"""PyMilo RandomState(MT19937) object transporter."""
+import numpy as np
+from ..utils.util import is_primitive, check_str_in_iterable
+from .transporter import AbstractTransporter
+
+# Handling Numpy's RandomState
+
+
+class RandomStateTransporter(AbstractTransporter):
+    """Customized PyMilo Transporter developed to handle RandomState field."""
+
+    def serialize(self, data, key, model_type):
+        """
+        Serialize instances of the RandomState class.
+
+        Record the `state` associated fields of random state object.
+
+        :param data: the internal data dictionary of the given model
+        :type data: dict
+        :param key: the special key of the data param, which we're going to serialize its value(data[key])
+        :type key: object
+        :param model_type: the model type of the ML model, which its data dictionary is given as the data param.
+        :type model_type: str
+        :return: pymilo serialized output of data[key]
+        """
+        if (
+            (model_type == "MLPRegressor" and key == "_random_state")
+        ):
+            rng = data[key]
+            data[key] = {
+                'state': (
+                    rng.get_state()[0],
+                    rng.get_state()[1].tolist(),
+                    rng.get_state()[2],
+                    rng.get_state()[3],
+                    rng.get_state()[4]
```

Comment:
It's a little bit odd. Can you make this part a little bit informative? like what's `rng`? why the second element need to be `tolist()`ed?